packages/io-devtools/src/stdio/printer.ts

```diff
@@ -63,20 +86,39 @@ export const printRecord = <TRecord extends object>(obj: TRecord, title?: string
  * the property name.
  *
  * @param {Record<string | number, unknown>} records
- * @param {string[]} [header]
+ * @param {string[]} [header] - header row if provided (OPTIONAL)
+ * @param center - center text if true (OPTIONAL)
+ * @param keyColor - color of keys in table (DEFAULT: WHITE)
+ * @param valueColor - color of values in table (DEFAULT: MAGENTA)
```

Comment:
Here the JSDoc should be something like:

```typescript
/**
 * @param {boolean} [center] ...
 * @param {Preprocess} [keyColor] ...
 */
```

The `[]` mean a param is optional and the `{}` denote the type